devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -300,6 +303,22 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	def checkVal
+	// the last message sent by configure is a configuration get, so if we get a report, we succeeded in transmission
+	// and if the parameter 1 values match what we expect, then the configuration probably succeeded
+	if (isZipato()) {
+		checkVal = 3
+	} else if (isYale()) {
+		checkVal = state.alarmLength
+	}
+	if (checkVal) {
+		state.configured = (checkVal == cmd.scaledConfigurationValue)
+	} else {
+		state.configured = true
```

Comment:
This is still set to true?

Reply:
yeah because if there's no checkVal there's nothing to configure.